openapi.yaml

```diff
@@ -958,296 +1032,476 @@ definitions:
+        properties:
+          type:
+            $ref: '#/definitions/JobType'
+          attributes:
+            $ref: '#/definitions/JobPostAttributes'
+  JobPatchBody:
+    type: object
+    properties:
+      data:
+        type: object
+        properties:
+          type:
+            $ref: '#/definitions/JobType'
+          attributes:
+            allOf:
+              - $ref: '#/definitions/JobPatchAttributes'
+              - $ref: '#/definitions/JobPostAttributes'
+  SsnPostAttributes:
+    type: object
+    required:
+      - ssn
+    properties:
+      ssn:
+        type: string
+        description: 9 digits SSN
+        example: '123456789'
+  SsnAttributes:
+    type: object
+    properties:
+      ssnStatus:
+        type: string
+        description: SSN status
+        enum:
+          - vault
+          - valid
+          - invalid
+  SsnPostBody:
+    type: object
+    properties:
+      data:
+        type: object
+        properties:
+          type:
+            $ref: '#/definitions/SsnType'
+          attributes:
+            $ref: '#/definitions/SsnPostAttributes'
+  SsnResource:
+    type: object
     properties:
       id:
         type: string
-        description: Meal plan ID
+        description: 9 digits SSN
+        example: '123456789'
       type:
-        type: string
-        example: meal-plans
+        $ref: '#/definitions/SsnType'
       links:
-          $ref: '#/definitions/SelfLink'
+        $ref: '#/definitions/SelfLink'
       attributes:
-        properties:
-          mealPlan:
-            type: string
-            description: Description of meal plan
-            example: Orange Rewards
-          balance:
-            type: number
-            format: float
-            description: Remaining balance on meal plan in USD
-            example: 23.8
-          lastUsedDate:
-            type: string
-            format: date-time
-            description: The last time the meal plan was used
-            example: '2017-07-14T16:45:15Z'
-          lastUsedLocation:
-            type: string
-            description: Description of place the meal plan was last used
-            example: 'Java II'
-  NewAddressResultObject:
+          $ref: '#/definitions/SsnAttributes'
+  SsnResult:
+    type: object
     properties:
+      links:
+        $ref: '#/definitions/SelfLink'
       data:
-        $ref: '#/definitions/NewAddressResourceObject'
-  NewAddressResourceObject:
+        $ref: '#/definitions/SsnResource'
+  MealPlanAttributes:
+    type: object
     properties:
+      mealPlan:
+        type: string
+        description: Description of meal plan
+        example: Orange Rewards
+      balance:
+        type: number
+        format: float
+        description: Remaining balance on meal plan in USD.
+        example: 3.50
+      lastUsedDate:
+        type: string
+        format: date-time
+        description: The last time the meal plan was used.
+        example: '2017-07-14T16:45:15Z'
```

Comment:
I guess I will rename this to `lastUsedDateTime`. Also, do you think it will be handy if we have a filter for this? Something like `filter[lastUsedDate][before]=2017-07-14T16:45:15Z`? (I'm not sure for the operation though)